contracts/DrawCalculator.sol

```diff
@@ -360,11 +360,12 @@ contract DrawCalculator is IDrawCalculator, Ownable {
         uint256[] memory masks = new uint256[](_prizeDistribution.matchCardinality);
         uint256 _bitRangeMaskValue = (2**_prizeDistribution.bitRangeSize) - 1; // get a decimal representation of bitRangeSize
 
-        for (uint8 maskIndex = 0; maskIndex < _prizeDistribution.matchCardinality; maskIndex++) {
-            // create mask of width bitRangeSize bits at index
-            uint256 _matchIndexOffset = uint256(maskIndex) * uint256(_prizeDistribution.bitRangeSize);
+        if(_prizeDistribution.matchCardinality == 0) return masks;
```

Comment:
```suggestion
        if (_prizeDistribution.matchCardinality == 0) { 
            return masks;
         }
```

Reply:
matchCardinality cannot be 0 so removing this